packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts

```diff
@@ -1,4 +1,6 @@
 /**
+ * @deprecated Use `areArraysEqual` instead.
+ *
```

Comment:
I thought that we can have something more common, and not only for string arrays